cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -32,14 +32,7 @@
         {% elif block.block_type == 'feedback' %}
             {{- form_block.render(block, 'content', loop.index0) -}}
         {% elif 'post_preview_snapshot' in block.block_type %}
-            <div class="block
-                        {{ block.block.meta.classname if block.block.meta.classname else '' }}">
```

Comment:
This second class doesn't ever seem to get used in practice; both Sublanding pages that use `PostPreviewSnapshot` have `<div class="block ">`. @anselmbradford @Scotchester should something else go here [in its new location](https://github.com/cfpb/cfgov-refresh/blob/98b59e44bdb6783005dc0f87f30e7c904dbad6f3/cfgov/jinja2/v1/_includes/organisms/post-preview-snapshot.html#L22)?

Reply:
Hmm it appears [this](https://github.com/cfpb/cfgov-refresh/pull/1522) is where it was added. Maybe something to do with that layout where the breakout sidebar was contained in the same box as the text intro. I think it's fine to remove.

I noticed something unrelated looking at this. The text introduction has two `block__flush-top` classes added. Anyone see why that's the case?
<img width="1300" alt="screen shot 2019-01-30 at 10 17 30 am" src="https://user-images.githubusercontent.com/704760/51991083-5c9d5100-2478-11e9-8ab0-9a2a7c778074.png">

Reply:
@anselmbradford it looks like that gets rendered in the Sublanding page template [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/sublanding-page/index.html#L10), which calls [this macro](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/templates/render_block.html). That renders `block__flush-top` twice, once [on this line](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/templates/render_block.html#L3) because it's the first element, and then again [on this line](https://github.com/cfpb/cfgov-refresh/blob/026ff4960ebb533e99b1b42335b4afd3b970c8b5/cfgov/jinja2/v1/_includes/templates/render_block.html#L12) because of [this `Meta.classname` definition](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/atomic_elements/molecules.py#L111).

Reply:
How best to fix that?

Reply:
As with all of these, I'd like to deprecate our use of `render_block` and move all of the StreamField blocks to using the Wagtail standard `{% include_block %}`, if we can. We could do that for `TextIntroduction` - does this actually cause a problem, or is it just unnecessary duplication?

Reply:
It's just unnecessary duplication, but yeah, I'd say it gives us added motivation to migrate `TextIntroduction` to `{% include_block %}`.